src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/LanguageServerFeatureOptions.cs

```diff
@@ -47,4 +47,9 @@ internal abstract class LanguageServerFeatureOptions
     /// character with a soft-selected item will not commit that item.
     /// </summary>
     public abstract bool SupportsSoftSelectionInCompletion { get; }
+
+    /// <summary>
+    /// Indicates that VSCode-compatible completion trigger character set should be used
+    /// </summary>
+    public abstract bool VsCodeCompletionTriggerCharacters { get; }
```

Comment:
Nit: Consider adding "Use" to the front of this, to match the existing flags (and make it more verb-y)